packages/components/src/dimension-control/stories/index.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * WordPress dependencies
+ */
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+/**
+ * Internal dependencies
+ */
+import { DimensionControl } from '..';
+import sizes from '../sizes';
+
+export default {
```

Comment:
Thank you for adding Storybook examples!

Reply:
✅ https://github.com/WordPress/gutenberg/pull/47351/commits/42a0ae01fa6fdfeb3e8eb3bc3526e6f54ad0101b